Incorrect arguments to `subprocess.CalledProcessError`

On Edison (which seems to link everything statically regardless of compiler flags), I am getting this error:

```
[bzip2] running ['hit', 'build-postprocess', u'--shebang=multiline', u'--write-protect', u'--relative-sh-script=/bin/.*-config', u'--relative-pkgconfig', u'--relative-rpath', u'--relative-symlinks', u'--relative-symlinks', u'--check-relocatable', u'--check-ignore=.*\\.pyc$', u'--check-ignore=.*\\.pyo$']
[ERROR] [u'/global/u1/m/memmett/.hashdist/bld/patchelf/3p5j3zzm6egh/bin/patchelf', '--shrink-rpath', u'/global/u1/m/memmett/.hashdist/bld/bzip2/ibugkmd3byfn/bin/bzip2recover'] failed: 1
[bzip2|ERROR] hit command failed: __init__() takes at least 3 arguments (2 given)
```

Note that 'bzip2' is the first relocatable package that is compiled when I start a fresh stack.

--- a/hashdist/core/build_tools.py
+++ b/hashdist/core/build_tools.py
@@ -188,7 +188,7 @@ def _check_call(logger, cmd):
     out, err = p.communicate()
     if p.wait() != 0:
         logger.error('%r failed: %d' % (cmd, p.wait()))
-        raise subprocess.CalledProcessError(
+        raise Exception(
             'Command %r failed with code %d and stderr:\n%s' % (cmd, p.wait(), err))
     return out
 
@@ -594,5 +594,3 @@ def add_modelines(scriptlines, language):
     else:
         vi_modeline = []
     return shebang + emacs_modeline + body + vi_modeline
-
-

